### PR TITLE
update command names

### PR DESCRIPTION
**Pull request by alexherbo2**

### CODE REVIEW

**Comment by casimir** [comment]
I would advise against changing all files at once, this will create conflicts against all currently opened PRs. It would be a better approach to do it step by step, encouraging people to do it in their respective PR.

**Comment by alexherbo2** [comment]
I was thinking it would be the right time to change all that in once, as there is no merge conflict against the other pull requests (at the time being).

**Comment by casimir** [comment]
I didn't check. I agree on the timing.

File: README.asciidoc [hunk 1]
```diff
@@ -1185,24 +1185,24 @@ Manipulation of the displayed text is done through highlighters, which can be ad
 or removed with the command
 
 -----------------------------------------------------
```
Comment (mawww) [inline README.asciidoc]: Please fix these lines as well so that they match the length of the content.

Comment (alexherbo2) [inline README.asciidoc]: Done for all docs, but I did not touch tabs that could mess with the alignment.

**Pull request merged**